packages/instantsearch.css/.stylelintrc.json

```
@@ -0,0 +1,25 @@
+{
+  "extends": [
+    "stylelint-prettier/recommended",
+    "stylelint-config-standard-scss",
+    "stylelint-config-prettier-scss",
+    "stylelint-a11y/recommended"
+  ],
+  "plugins": ["stylelint-no-unsupported-browser-features", "stylelint-order"],
+  "rules": {
+    "selector-class-pattern": "(?:^ais-[A-Z][a-zA-Z0-9]+[-a-zA-Z0-9]+$|^rheostat(?:-[a-z]+)?$)",
+    "selector-pseudo-element-no-unknown": [
+      true,
+      {
+        "ignorePseudoElements": ["input-placeholder"]
+      }
+    ],
+    "plugin/no-unsupported-browser-features": [
+      true,
+      {
+        "browsers": ["defaults"],
```

We don't have one now, but later on we can use a `.browserlistrc` file to share between Stylelint, Babel, etc.

---

We should explicitly declare Internet Explorer 11 support as `defaults` doesn't include it.

```suggestion
        "browsers": ["defaults", "ie 11"],
```